# flag true means pause, flag false means unpause/resume
bool pause

# paused/resumed task
string commanded_task
